src/test/ledger/Invariants_test.cpp

```diff
@@ -55,6 +55,7 @@ class Invariants_test : public beast::unit_test::suite
     // changes that will cause the check to fail.
     void
     doInvariantCheck( bool enabled,
+        std::vector<std::string> const& expect_logs,
```

Comment:
right - the test is now also checking some aspect of the logged failure. If this turns out to be a maintenance burden, we can disable/remove the log message check.